pkg/security/admission/scc_exec_test.go

```diff
@@ -110,10 +110,5 @@ func TestExecAdmit(t *testing.T) {
 			t.Errorf("%s: no actions found", k)
 		}
 
-		if v.shouldHaveClientAction {
```

Comment:
> is there a test making sure exec works if the service account had rights and the user didn't?

Yes, its a delegation, so since we no longer clear and we test that we plumb through, the original admission tests get it.